wiki/Performance_Troubleshooting/en.md

```diff
@@ -86,3 +86,22 @@ While this may seem like the best option (and indeed can be, if your hardware ca
 - GPUs generally aren’t meant to render 1000+ frames per second, so unexpected behaviour may occur, including lower performance or higher latency than otherwise attainable.
 
 Even so, if heat generation and potentially overworking your hardware is not considered an issue, unlimited may be the option that provides you the most stable frame times.
+
+### Custom
+
+This frame limiter attempts to keep a frame rate equal to your custom preference. However, this is a setting that cannot be changed to within the in-game menu.
+
+To change osu! custom maximum framerate, you'll need to tweak some values in the config file. Here's the steps:
+
+- Open your osu! folder. (The default is `C:\Users\<yourPCusername>\AppData\Roaming\osu!`.)
+- Open the `osu!.<yourPCusername>.cfg` file with Notepad or any text editors.
+- Find the line `FrameSync = <value>` and change the value to `Custom`.
+- Find the line `CustomFrameLimit = <value>` and change the value to your preference.
```

Comment:
```suggestion
1. Open your osu! folder. (The default is `C:\Users\<yourPCusername>\AppData\Roaming\osu!` for Windows; `/Applications/osu!.app/Contents/Resources/drive_c/Program Files/osu!/` for Mac.)
2. Open the `osu!.<yourPCusername>.cfg` file with Notepad or a text editor of your choice.
3. Find the line `FrameSync = <value>` and change the value to `Custom`.
4. Find the line `CustomFrameLimit = <value>` and change the value to your preference.
```
not sure about the full path for mac but needs to be there regardless

Reply:
wouldn't it be quicker to refer to the `Open osu! folder`-button in the options?

Reply:
That's also an option

Reply:
I'll check the wiki for the correct path for Mac: `/Applications/osu!.app/Contents/Resources/drive_c/osu!`
I've also change `Roaming` to `Local` in Windows path.
And thanks for your suggestion about the `Open osu! folder` in-game.